source/git-parser.js

```diff
@@ -162,7 +162,7 @@ exports.parseGitLog = (data) => {
       parser = parseFileChanges;
       return;
     }
-    if (rows[index + 1] && rows[index + 1].indexOf('\x00commit ') == 0) {
+    if (rows[index + 1] && rows[index + 1].match(/^\u0000+commit/) != null) {
```

Comment:
Would `test` instead of `match` also work here?

Reply:
Indeed it will be better, I'll update the PR